travel-event-host-app/src/components/searchSection/searchSection.tsx

```diff
@@ -25,11 +27,24 @@ export default function SearchSection({ keyword }: { keyword: string }) {
 
   useEffect(() => {
     const fetch = async () => {
-      const eventsResultFetch = await getEventsBySearchQuery(keyword, categories);
+      const eventsResultFetch = await getEventsBySearchQuery(
+        keyword,
+        getCheckedCategories(categoryCheckboxState),
```

Comment:
pass Categories[] to the method. But you will need to convert your checkbox state into Category[]